core/src/com/unciv/ui/popups/options/GameplayTab.kt

```diff
@@ -42,6 +42,11 @@ fun gameplayTab(
         "Automated units move on turn start",
         settings.automatedUnitsMoveOnTurnStart, true
     ) { settings.automatedUnitsMoveOnTurnStart = it }
+    optionsPopup.addCheckbox(
+        this,
+        "Automated units can upgrade",
```

Comment:
This text needs to be added to template.properties for translation

Reply:
Does this line now need the tr() function for translation ?

        "Automated units can upgrade".tr(),